torchtext/data/datasets_utils.py

```diff
@@ -318,3 +319,22 @@ def pos(self):
 
     def __str__(self):
         return self.description
+
+
+class _ParseSQuADQAData(IterDataPipe):
```

Comment:
Can we add a short docstring with an explanation of what this function does and why it's put here?  It would probably be good to mention that this util function is relevant to both SQuAD1 and SQuAD2 datasets.